notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install pyglet
</pre><p>If this is required, it should be moved to the dependencies list for the relevant environment so that it doesn't need to be installed separately</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>